<!---->
<launch>

<node pkg="laser_filters" type="scan_to_cloud_filter_chain" output="screen" name="cloud_filter_front">
<remap from="scan" to="/lms1xx_front/scan" />
<remap from="cloud_filtered" to="front_cloud_filtered"/>
<param name="high_fidelity" value="true" />
<param name="target_frame" type="string" value="lms1xx_front" />
</node>

<!-- apply laser filter for front scan
<node pkg="PCL_test" type="outlier_removal" output="screen" name="test">

<param name="cloud_in" value="/front_cloud_filtered" />
<param name="cloud_out" value="/outlier_filter" />
<param name="radius" value="0.1" />
<param name="neigbour_nr" value="50" />
<param name="distance" value="0" />

</node>-->
<!-- save to pcd file-->
<node pkg="pcl_test" type="pointcloud_to_pcd2" output="screen" name="convert">

<remap from="input" to="front_cloud_filtered"/>
</node>
</launch>
